packages/react/ds-core-form/src/ui/Field/types.ts

```diff
@@ -19,19 +24,16 @@ export enum InputType {
   // Color = "color",
 }
 
-export interface FieldProps {
-  /* A unique identifier for the Field */
-  id?: string;
-  /* Additional CSS classes */
-  className?: string;
-  /* Child elements */
-  children?: React.ReactNode;
-  /* Inline styles */
-  style?: React.CSSProperties;
+export type UnionInputProps = CheckboxProps | TextProps | TextareaProps;
```

Comment:
Generally, I'd try to avoid describing the type itself in the type name. "Union" is a set theory descriptor of the type itself, which seems a bit semantically clunky. 

Might we be able to use something simpler, like `InputProps`?